examples/kuesa/assets/models/damagedhelmet/rotating_glTF/helmet.qrc

```diff
@@ -0,0 +1,6 @@
+<RCC>
```

Comment:
do we need all the other files if we're only going to use these 2?

Reply:
Was just allowing the choice until we had come to a decision. It does seem like the built in animation is the way to go, so I can remove the other versions.